Documentation/quickstart/index.rst

```diff
@@ -1,16 +1,18 @@
 .. todo::
-  This should be a brief introduction to how NuttX works and how to get it running. A widely used board could be targeted to create a brief demonstration on how to clone, configure, build, flash and run NuttX.
-  For now, the content that is here presented is what remains of the old Porting Guide which is not already in another section of this documentation.
 
 ==========
 Quickstart
```

Comment:
I would change this to "Getting Started" since it reflects the complete content you added. Then, "Quickstart" is indeed a fast way to get started and remaining section go into detail over each step.

Reply:
@v01d Great, will change the title.